### PR TITLE
Restructuring error handling

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: server/gokbg3/grails-app/views/error/error.gsp [hunk 1]
```diff
@@ -7,18 +7,35 @@
...
+        </h2>
+    </div>
 
     <p><strong>${request.forwardURI}</strong></p>
-
+    <br />
+    <br />
     <g:if test="${exception}">
-        <p>${exception.message}</p>
+        <p>${exception?.message}</p>
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/views/error/error.gsp]: see above - null check necessary indeed?

**Review by croqueGrec09 — COMMENTED** [review]
one remark

**Comment by Moetez16** [comment]
Update PR

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**